### ISSUE TITLE
engine.say has no sound after not using window about 1 hour

### ISSUE THREAD

**Issue by enxuan** [nateshmbhat/pyttsx3#273]
I have code to say tts:
  import pyttsx3
  engine = pyttsx3.init()
  engine.setProperty('rate', rate)
  engine.setProperty('volume', volume)
  engine.say('I will speak this text')
  engine.runAndWait()
This code work ok if my PC is using or other PC remote to my PC.
But if my PC is not using about 1 hour (no touch, no any event), if I run jenkins build from jenkins PC (not my PC), it has no sound at all.
I am using window 10, already set never sleep, fixed speaker for speech to text setting.
lib version: 2.90

**Comment by Jiangshan00001** [comment]
"But if my PC is not using about 1 hour (no touch, no any event), if I run jenkins build from jenkins PC (not my PC), it has no sound at all."
the code is executed in my PC？
the code is executed using which mode？ ssh？ vnc? remote desktop? or locally？

**Comment by enxuan** [comment]
"But if my PC is not using about 1 hour (no touch, no any event), if I run jenkins build from jenkins PC (not my PC), it has no sound at all." ==> it means that we do not touch to "my PC" and the source code is auto get to "my PC" and run on it.
the code is executed in my PC？ --> yes, it is, the code is auto sync by jenkins and run on my PC
the code is executed using which mode？ ssh？ vnc? remote desktop? or locally？ --> the code is run on my PC so it run locally.
If when code is running, I use other PC to remote to "my PC", I can hear engine.say speak normally. If I close the remote, "my PC" will has no sound at all.
If when code is running, I log in to "my PC" and has event, the sound play normally again.

**Comment by enxuan** [comment]
yes, I think I have same issue with https://github.com/nateshmbhat/pyttsx3/issues/263
The issue seem due to engine.say() has no sound when PC screen is locked.
You can try to reproduce it by run you code, after run press window+L to off the screen and check if you can hear the sound or not.
